cmd/svcat/plan/get_cmd.go

```diff
@@ -112,10 +112,6 @@ func (c *getCmd) Validate(args []string) error {
 }
 
 func (c *getCmd) Run() error {
-	fmt.Println("KUBENAME: ", c.kubeName)
```

Comment:
I saw this output in my last built svcat - glad you are removing this, I wasn't aware we had actually checked this in like this.